.github/SUPPORT.md

```diff
@@ -0,0 +1,20 @@
+# Jekyll Support
+
+## Getting Help
+
+**Jekyll's issue tracker is not a support forum.**
+
+If you're looking for support for Jekyll, there are a lot of options:
+
+* Read [Jekyll Documentation](https://jekyllrb.com/docs/home/)
+* If you have a question about using Jekyll, start a discussion on [Jekyll Forum](https://talk.jekyllrb.com/) or [StackOverflow](https://stackoverflow.com/questions/tagged/jekyll)
+* Chat with Jekyllers &mdash; Join [our Gitter channel](https://gitter.im/jekyll/jekyll) or [our IRC channel on Freenode](irc:irc.freenode.net/jekyll)
+
+There are a bunch of helpful community members on these services that should be willing to point you in the right direction.
+
+## Report a bug
+
+* If you think you've found a bug within a Jekyll plugin, open an issue in that plugin's repository.
```

Comment:
How do they know where to find the repository? Generally https://rubygems.org/gem/plugin-name then click `Homepage` is what I do personally. We should help the user a little here.